From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke@hauke-m.de>
Date: Sat, 25 Oct 2025 15:38:31 +0200
Subject: Remove ubifs test builds

The build from the tar file fails with:
```
make[3]: *** No rule to make target 'tests/ubifs_tools-tests/images/good.gz', needed by 'all-am'.  Stop.
```

The tar does not contain the test data files. We do not need these
tests, just deactivate them.

A new tar file created with this upstream fix should contain the missing
files:
https://patchwork.ozlabs.org/project/linux-mtd/patch/20250220112554.2150046-1-m.olbrich@pengutronix.de/
---
 Makefile.am | 1 -
 1 file changed, 1 deletion(-)

--- a/Makefile.am
+++ b/Makefile.am
@@ -64,7 +64,6 @@ include tests/jittertest/Makemodule.am
 include tests/checkfs/Makemodule.am
 include tests/fs-tests/Makemodule.am
 include tests/mtd-tests/Makemodule.am
-include tests/ubifs_tools-tests/Makemodule.am
 endif
 
 if UNIT_TESTS
